openmdao/visualization/n2_viewer/index.html

```diff
@@ -1,12 +1,12 @@
-<div id="all_pt_n2_content_div">
+<div id="all_pt_n2_content_div" class="non-embedded-n2">
     <div id="ptN2ContentDivId">
         <!-- The Model -->
         {{help}}
 
         <!-- Toolbar -->
         <div>
             <div id="toolbarLoc">
-                <img src="../../assets/logo.png" id="company-logo" />
+                <!-- img src="../../assets/logo.png" id="company-logo" / -->
```

Comment:
could we clean up the old commented code?

Reply:
Removed the one from embed_n2.py, the one in index.html will get fixed on the refactor PR